uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -427,6 +434,46 @@ private static function getNotificationReceiversIDs($typeId) {
   }
 
   /**
+   * Checks, based on the $params array, if the Public Holiday Leave Requests
```

Comment:
In English (and in Portuguese as well) we should use commas to separate nonessential phrases in a sentence. 

See that I can remove the "based on the $params array" entirely and the sentence still makes sense: "Checks if the Public Holiday Leave Requests must be updated after saving the Absence Type". So, since that part is nonessential, commas should be used (Just used them again here :D).

For reference, check rules 5 and 6 here: http://www.grammarbook.com/punctuation/commas.asp